content/blog/2021-10-11-1.6.0-what-you-need-to-know.en.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Cryptomator 1.6.0: What you need to know"
+slug: 1-6-0-what-you-need-to-know 
+date: 2021-10-11
+tags: [cryptomator, windows, mac, linux]
+
+summary: "In this blog post, we'd like to give you some news about the upcoming major update of Cryptomator to version 1.6.0."
+---
+
+Hello Community!
+
+In this blog post, we'd like to give you some news about the upcoming major update of Cryptomator to version 1.6.0.
+We'll be highlighting the most significant changes and new features and make sure that you are ready for the update.
+
+## Changes and Features
+The two most important changes are the usage of a new vault format (version 8) and a long promised integration of the [Sanitizer](https://community.cryptomator.org/t/sanitizer-how-to-use/43) (now called Vault Health Check).
+For a more complete list, read the [release page](https://github.com/cryptomator/cryptomator/releases/tag/1.6.0-rc1) of Cryptomator.
+
+### Auto Lock
+A feature already wished in its earliest days will be present: Auto Lock – the automatic locking of a vault.
+For every vault you can set up an idle timer after which the vault is automatically locked.
+If any write or read happens during the time span, the timer is reset.
+
+### Redesigned Error Dialog
+After a lot of indirect feedback from you about the error dialog, we decided to change its design to fit more of your needs.
+The most obvious and important change is the new error code.
+It might be as cryptic as the already existing stack trace, but it speeds up the search for solutions or workarounds for your specific problem in our error code database.
+Along with the error code the dialog also provides links to quickly query the database.
+And if the error is not yet known, it'll make it easier for you to report it in a format that helps us understand the problem.
+
+### Vault Format 8
+The big change behind the scenes is a new vault format.
+Starting with 1.6.0, it will be used by default and _enforced_.
+The new format prepares Cryptomator for future features and corrects inconsistencies in former versions.
+For more details, check out the more-in-depth [article](https://cryptomator.org/blog/2021/10/11/vault-format-8/) about it.
+
+### Vault Health Check
+We added an integrated tool to detect and fix structural problems of a vault (e.g., missing directories).
+Until Cryptomator 1.5.0, this task was done by the so-called [Sanitizer] (https://github.com/cryptomator/sanitizer). But the tool was hard to maintain and hard to use so that it  was abandoned with the goal to integrate similar functionality directly within Cryptomator.
```

Comment:
```suggestion
Until Cryptomator 1.5.0, this task was done by the so-called [Sanitizer] (https://github.com/cryptomator/sanitizer). But the tool was hard to maintain and hard to use so that it was abandoned with the goal to integrate similar functionality directly within Cryptomator.
```